package.json

```diff
@@ -63,20 +66,34 @@
   },
   "config": {
     "commitizen": {
-      "path": "cz-conventional-changelog"
+      "path": "node_modules/cz-conventional-changelog"
     }
   },
   "commitlint": {
     "extends": [
       "@commitlint/config-conventional"
-    ]
+    ],
+    "rules": {
+      "footer-max-line-length": [
+        2,
+        "always",
+        500
+      ]
+    }
+  },
+  "lint-staged": {
+    "*.{js,vue,ts,tsx}": "npm run lint"
   },
-  "release": {
-    "branches": [
-      "main"
-    ]
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
   },
   "dependencies": {
+    "@semantic-release/commit-analyzer": "^10.0.1",
```

Comment:
Why do we need these dependencies as regular dependencies? Wouldn't it make more sense to have them as dev dependencies?

Reply:
you are right, thanks